Mathlib/Topology/UniformSpace/CompactConvergence.lean

```diff
@@ -412,9 +411,9 @@ compact space.
 For non-T2 spaces, the assumption `LocallyCompactSpace α` is stronger than we need and in fact
 the `←` direction is true unconditionally. See `ContinuousMap.tendstoLocallyUniformly_of_tendsto`
 and `ContinuousMap.tendsto_of_tendstoLocallyUniformly` for versions requiring weaker hypotheses. -/
```

Comment:
Could you fix this docstring? The assumption is now exactly the right one for the `→` direction, right?